packages/language-server/src/ls-config.ts

```diff
@@ -45,6 +45,7 @@ const defaultLSConfig: LSConfig = {
     },
     svelte: {
         enable: true,
+        useNewTransformation: false,
```

Comment:
This is used to determine whether or not to use the new transformation. My idea is to go into a month-long beta phase where this is `false` by default and we encourage people to switch it on to try it out. After that we toggle it to `true` by default and then see what the feedback is. If everything is good, we can remove it ~2 months afterwards and remove all the old code, simplifying the code base.

This boolean is used in a few places in some Providers to do something differently.